HLT/BASE/CMakeLists.txt

```diff
@@ -149,6 +149,13 @@ get_directory_property(incdirs INCLUDE_DIRECTORIES)
 generate_dictionary("${MODULE}" "${MODULE}LinkDef.h" "${HDRS}" "${incdirs}")
 
 set(ROOT_DEPENDENCIES Core Geom Graf Hist MathCore Net RIO Tree XMLParser)
+
+if(${ROOT_VERSION_MAJOR} LESS 6)
+	list(APPEND ROOT_DEPENDENCIES Cint)
+else()
+	list(APPEND ROOT_DEPENDENCIES Cling)
+endif(${ROOT_VERSION_MAJOR} LESS 6)
```

Comment:
Can you turn it to a simple `endif()` please? Otherwise it looks good to me.

Reply:
Also, use spaces not tabs.

Reply:
Thanks, fixed!